example/conf/default.yaml

```diff
@@ -1,6 +1,24 @@
 testValue: 123
 mongo:
   db: 'rapptor-example'
+logging:
+  reporters:
+    hipchat:
+      - 
+        auth_token: ''
+        room: ''
+        roomToken: ''
+        debug: true
+        notify: false
+        events:
```

Comment:
I think that'd be fine, problem is the way its loaded. If we have the hipchat reporter there the module will be loaded. Maybe have an enable flag?